lib/interface/login/login.dart

```diff
@@ -49,22 +49,33 @@ class LoginScreenState extends State<LoginScreen>
 
   @override
   onAuthStateChanged(AuthState state) {
-    if(state == AuthState.LOGGED_IN)
+    if (state == AuthState.LOGGED_IN)
       Navigator.of(_ctx).pushReplacementNamed("/home");
   }
+
   var _value = "en";
   DropdownButton _normalDown() => DropdownButton<String>(
+        underline: Container(
+          height: 2,
+          color: Colors.lightBlue,
+        ),
         items: [
           DropdownMenuItem(
             value: "en",
-            child: Text(
-              "English",
+            child: Container(
+              width: MediaQuery.of(context).size.width * 0.5,
+              child: Text(
+                "English",
+              ),
             ),
           ),
           DropdownMenuItem(
             value: "ar",
-            child: Text(
-              "Arabic",
+            child: Container(
+              width: MediaQuery.of(context).size.width * 0.5,
```

Comment:
Using an aspect ratio based design or a scaling factor based design would be better